cpg_utils/hail.py

```diff
@@ -187,3 +187,33 @@ def image_path(suffix: str) -> str:
     prefix = os.getenv('CPG_IMAGE_REGISTRY_PREFIX')
     assert prefix
     return f'{prefix}/{suffix}'
+
+
+def authenticate_job(
```

Comment:
A small suggestion to make this clearer as we transition to other clouds

```suggestion
def authenticate_cloud_credentials_in_job(
```